src/async_impl/client.rs

```diff
@@ -237,6 +239,15 @@ impl ClientBuilder {
         }
         self
     }
+
+    /// Set number of DNS threads.
+    #[inline]
+    pub fn threads(&mut self, threads: usize) -> &mut ClientBuilder {
```

Comment:
Perhaps to be clear that it's only for DNS, and not something else, this could be `dns_threads`?